README-Senn.md

```diff
@@ -0,0 +1,72 @@
+# Senn6x
+
+Senn6x is an experimental Residual Channel Attention Network designed for Image Super-Resolution.
+
+![Little Guy](https://github.com/Senn1/technical-writing-test-repo/blob/master/Figures/littleguy.png)
```

Comment:
This will always link back to your original repo, you want to add a relative link here, not an absolute one.
See https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files